lib/src/common/repos_service.dart

```diff
@@ -817,20 +831,23 @@ class RepositoriesService extends Service {
     );
   }
 
+  /// Adds a deploy key for a repository by passing in the title and key as strings.
+  ///
+  /// API docs: https://developer.github.com/v3/repos/keys/#create
+  Future<PublicKey> createDeployKeyFromStrings(RepositorySlug slug,
```

Comment:
Thoughts on this name? It's just meh with me. Or we could just drop it and have people continue to
`createDeployKey(slug, CreatePublicKey(title, key))`